websites/migrations/0046_filetype_to_resourcetype.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.1.14 on 2022-04-25 22:37
+
+from django.db import migrations
+
+from websites.constants import WEBSITE_SOURCE_STUDIO
+
+
+def migrate_fields_forward(apps, schema_editor):
+    migrate_field(apps, True)
+
+
+def migrate_fields_backward(apps, schema_editor):
+    migrate_field(apps, False)
+
+
+def migrate_field(apps, forward):
+    Website = apps.get_model("websites", "Website")
+    WebsiteContent = apps.get_model("websites", "WebsiteContent")
+    original_name = "filetype" if forward else "resourcetype"
+    updated_name = "resourcetype" if forward else "filetype"
+    try:
+        ocw_www = Website.objects.get(name="ocw-www")
```

Comment:
Since this will only fetch a website having name `ocw-www`, so I was wondering whether there's a need to update the ocw-www test sites too on RC? 
<img width="451" alt="image" src="https://user-images.githubusercontent.com/93309234/165384418-9ba154f8-2def-4360-9ef8-37e53d5e8e06.png">

Reply:
This a good catch, probably not a big deal but we should migrate all the data if we can.  I changed it to iterate all sites that use the `ocw-www` starter instead and this is ready for another look.

Reply:
It looks like all ocw-www websites are not using the same starter, some of them have different starters with a different name but I agree that it's not a big deal so as far as we have `ocw-www` named starter on production and `ocw-www` website is using that starter, this should be fine. 

However, if you feel that we should migrate all data on RC too then maybe we can use `Like` query (equivalent to `contains` in django ORM), something like this:

`ocw_www_sites = Website.objects.filter(name__contains="ocw-www")` 

and I think it will be worth checking that this matching string: `ocw-www` is not there in any course name. (we don't have such a course name on RC though).

Apart from this, some tests are failing on this PR, I'm not sure why, I tried re-running them but it didn't work.

Reply:
@MAbdurrehman12 Thanks for taking another look.  I fixed the issue with the tests and the backward migration argument.  I think for this case, only addressing sites with literally `ocw-www` as the starter makes sense, and all other starters are out of scope.  This PR is in response to changing the fields in the official `ocw-www` starter as it's defined in `ocw-hugo-projects`, and any other starters that have been modified outside of that context don't matter in this case.

Reply:
Sounds good!